spec/acceptance/init_spec.rb

```diff
@@ -39,6 +39,10 @@
   default_pam_pkcs11_conf = File.open(File.join(fixture_path, 'default_files', os_files_path, 'pam_pkcs11.conf')).read
   default_pkcs11_eventmgr_conf = File.open(File.join(fixture_path, 'default_files', os_files_path, 'pkcs11_eventmgr.conf')).read
 
+  # if fact('os')['family'] == 'Debian'
+  #   default_pam_config_pkcs11 = File.open(File.join(fixture_path, 'default_files', os_files_path, 'pam-config.conf')).read
+  # end
+
```

Comment:
Is this intentional?

Reply:
no. good catch, forgot to uncomment that. This `acceptance` stuff is currently unused, since it belongs to the beaker testing, which currently does not work.